patterns/1-initial/extensions-to-manage-contributions-at-scale.md

```diff
@@ -10,14 +10,29 @@ An InnerSource project is receiving too many contributions, making maintenance d
 
 As the number of new feature contributions to a mature InnerSource repository rapidly increases, it adds more burden on code reviews and maintenance. How do we allow for faster release of new features, encouraging innovation and experimentation; while also keeping the repository well maintained?
 
+## Story
+
+There is a strategic project that aims to collect the best innovations within a domain space to one common stack, promoting reuse of a common infrastructure and ensuring a standard user experience. Through InnerSource, various teams in the organization that work within the domain space get an opportunity to collaborate and contribute their innovations to the common codebase, which also offers more opportunities for their work to get adopted. However, a large number of contributions in parallel from several developers is making maintenance of the codebase difficult. This is adding a huge burden on the core team of maintainers who assume ownership over the code quality standards and enable the community through various forms of communication.
+
+Maintainers are burnt out due to:
+
+- Everlasting backlog of pull requests that need to be reviewed.
+- Job dissatisfaction: Majority of maintainers' time spent in community support leaves no room for innovation.
```

Comment:
I think this should be `room for innovation or development from from the maintainers` or something similar - since contributors outside of the maintainer group is technically innovation for the project

Reply:
hmm since the para is talking about maintainers I think it is implied? And it is talking about maintainers' job dissatisfaction?